Add readme for notebook examples (#733)
intel/scikit-learn-intelex: examples/notebooks/README.md

# Intel(R) Extension for Scikit-learn* notebooks :snake:

> This folder contains examples of jupyter notebooks which using Intel(R) extension for scikit-learn for known small datasets.  

Reviewer: ```suggestion
This folder contains examples of jupyter notebooks which using Intel(R) extension for scikit-learn for popular datasets.  
```

You need jupyter notebook to run these .ipynb files:

```bash

> conda install -c conda-forge notebook

Reviewer: And sklearnex

```  
or  
```bash
pip install notebook

> ```  

Reviewer: I think cool to add table with current examples and small description

Reviewer: For example
| **Algorithm** | **Workload** | **Notebook**|
|----------------|-----------------|----------------|
| **Logistic Regression** |  CIFAR-100 | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/logistictic_regression_cifar.ipynb) |
| **SVC**                         |  Adult         |     [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/svc_adult.ipynb) |
| ....                            | ....               | ....                |

# :snake: Intel(R) Extension for Scikit-learn* notebooks

> This folder contains examples of jupyter notebooks which using Intel(R) extension for scikit-learn for popular datasets.  

Reviewer 2: ```suggestion
This folder contains examples of Jupyter* notebooks that use Intel(R) extension for Scikit-learn for popular datasets.  
```

> You need jupyter notebook to run these .ipynb files:

Reviewer 2: ```suggestion
You need Jupyter* notebook to run the following .ipynb files:
```

> # Intel(R) Extension for Scikit-learn* notebooks :snake:

Reviewer: ```suggestion
#  :snake: Intel(R) Extension for Scikit-learn* notebooks
```

This folder contains examples of jupyter notebooks which using Intel(R) extension for scikit-learn for popular datasets.  
You need jupyter notebook to run these .ipynb files:

```bash
conda install -c conda-forge notebook scikit-learn-intelex
```  
or  
```bash
pip install notebook scikit-learn-intelex
```  

#### :pencil: Table of contents

| Algorithm               | Workload       | Notebook       | Sckit-learn estimator|
| :----------------------:| :------------: | :-------------:| :-------------------:|

> |    Logistic Regression  |   CIFAR-100  | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/logistictic_regression_cifar.ipynb)    | [sklearn.linear_model.LogisticRegression](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html) |

Reviewer: ```suggestion
|    LogisticRegression  |   CIFAR-100  | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/logistictic_regression_cifar.ipynb)    | [sklearn.linear_model.LogisticRegression](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html) |
```

Reviewer: and other - I think better merge

|          SVC            |     Adult    | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/svc_adult.ipynb) | [sklearn.svm.SVC](https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html) |
| K nearest neighbors classifier | MNIST | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/knn_mnist.ipynb) |    [sklearn.neighbors.KNeighborsClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html) |

> |        Nu SVR           | Medical charges | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/nusvr_medical_charges.ipynb) | [sklearn.svm.NuSVR](https://scikit-learn.org/stable/modules/generated/sklearn.svm.NuSVR.html) |

Reviewer: ```suggestion
|        NuSVR           | Medical charges | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/nusvr_medical_charges.ipynb) | [sklearn.svm.NuSVR](https://scikit-learn.org/stable/modules/generated/sklearn.svm.NuSVR.html) |
```
